snap/snapcraft.yaml

```diff
@@ -9,7 +9,16 @@ description: |
   For detailed description visit https://docs.magmacore.org/docs/next/lte/architecture_overview.
 
 grade: devel
-confinement: classic
+confinement: devmode
+
+apps:
+  magma-access-gateway-installer:
+    command: bin/install-agw
 
 parts:
-  magma-agw-installer:
+  magma-access-gateway-installer:
+    plugin: python
+    source: .
```

Comment:
All sources are integral part of this repository, so I figured it's fine to put '.', because I'd expect anyone building this snap to checkout the whole repository first. If my assumption is wrong, please correct me.